apps/wappush/test/unit/wappush_test.js

```diff
@@ -606,8 +652,12 @@ suite('WAP Push', function() {
         return WapPushManager.displayWapPushMessage(0);
       }).then(function() {
         done(function checks() {
```

Comment:
nit: it's easier to just do the checks in the "then" handler, and finish the promise chain by `.then(done, done)`